protected/gatk-tools-protected/src/main/java/org/broadinstitute/gatk/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -660,12 +661,8 @@ public void initialize() {
 
         vcfWriter.writeHeader(new VCFHeader(headerInfo, sampleSet));
 
-        try {
-            // fasta reference reader to supplement the edges of the reference sequence
-            referenceReader = new CachingIndexedFastaSequenceFile(getToolkit().getArguments().referenceFile);
-        } catch( FileNotFoundException e ) {
-            throw new UserException.CouldNotReadInputFile(getToolkit().getArguments().referenceFile, e);
-        }
+        // fasta reference reader to supplement the edges of the reference sequence
+        referenceReader = CachingIndexedFastaSequenceFile.checkAndCreate(getToolkit().getArguments().referenceFile);
```

Comment:
Why was the try catch removed? This method throws exceptions. Where are the exceptions handled?